.github/workflows/ci.yml

```diff
@@ -8,27 +8,17 @@ on:
 env:
   STRIPE_END_TO_END_TESTS_BACKEND_URL: ${{ secrets.STRIPE_END_TO_END_TESTS_BACKEND_URL }}
   STRIPE_END_TO_END_TESTS_PUBLISHABLE_KEY: ${{ secrets.STRIPE_END_TO_END_TESTS_PUBLISHABLE_KEY }}
+  GRADLE_OPTS: -Dkotlin.incremental=false
```

Comment:
Potentially a better way to do this - provide a CI `gradle.properties` file. In the `nowinandroid` project:

- https://github.com/android/nowinandroid/blob/main/.github/ci-gradle.properties
- https://github.com/android/nowinandroid/blob/main/.github/workflows/Build.yaml#L24